h3/src/proto/stream.rs

```diff
@@ -26,6 +26,7 @@ stream_types! {
     PUSH = 0x01,
     ENCODER = 0x02,
     DECODER = 0x03,
+    WEBTRANSPORT_BIDI = 0x41,
```

Comment:
I do not think that bi streams types should be here, as (from my understanding) this struct if for unidirectional stream types as defined in https://www.rfc-editor.org/rfc/rfc9114#section-11.2.4, whereas the `WEBTRANSPORT_BIDI` is a frame type for the HTTP/3 framing layer

Reply:
Ok, happy no to move it, where would you put it?